security/login_link.rst

```diff
@@ -796,3 +802,17 @@ features such as the locale used to generate the link::
 
         // ...
     }
+
+.. _customizing-link-lifetime:
+
+By default, generated links use :ref:`the lifetime configured globally <login-link-lifetime>`
+but you can change the lifetime per link using the third argument of the
+``createLoginLink()`` method::
+
+    // the third optional argument is the lifetime in seconds
+    $loginLinkDetails = $loginLinkHandler->createLoginLink($user, null, 60);
+    $loginLink = $loginLinkDetails->getUrl();
+
+.. versionadded:: 6.2
+
+    The option to customize the link lifetime was introduced in Symfony 6.2.
```

Comment:
```suggestion
    The parameter to customize the link lifetime was introduced in Symfony 6.2.
```
?

Reply:
Thanks for the review. At the end I changed this to `argument`.